framework/src/bluetooth/adaptation/bt-adatation-adapter.c

```diff
@@ -44,12 +45,16 @@ static void __bt_enable_cb(int err)
 
 int bt_adapt_enable(void)
 {
+	int ret;
```

Comment:
it's not critical, svace checks initialization of variable. so how about initialize it?